package.json

```diff
@@ -40,17 +40,19 @@
     "fix:eslint": "turbo --continue fix:eslint --",
     "fix:taplo": "taplo fmt",
     "fix:markdownlint": "markdownlint-cli2 --fix",
-    "fix:prettier": "prettier --write  --ignore-unknown .",
+    "fix:biome": "biome format --write",
```

Comment:
I wonder if it's worth us keeping `yarn fix:prettier` around as an alias for `yarn fix:biome`, for some period of time, as a convenience function (in case of muscle memory) while we still use the command in our other repos (both `blockprotocol` and internal).

Reply:
done in [08c9169](https://github.com/hashintel/hash/pull/6031/commits/08c9169eea68c8af7087fb1d3f842a420b87d952)

Reply:
Thanks!